test/multiple_ranges_test.cc

```diff
@@ -0,0 +1,14 @@
+#include "benchmark/benchmark.h"
+
+void BM_empty(benchmark::State& state) {
+  while (state.KeepRunning()) {
+    int product = state.range(0) * state.range(1) * state.range(2);
+    for (int x = 0; x < product; x++) {
```

Comment:
you should probably test that the product is what you expect.. or even better, that each range(x) is what you expect.